app/src/main/java/th/in/whs/k2ausbkbd/layout/KeyboardLayoutFactory.java

```diff
@@ -8,7 +8,8 @@
     private Map<String, Class<? extends Layout>> registry = new HashMap<>();
 
     private KeyboardLayoutFactory(){
-        register("QWERTY", QwertyLayout.class);
+        register("US QWERTY", USQwertyLayout.class);
```

Comment:
I'm not sure changing this would break backwards compatibility or not. Please test upgrading from the previous version, or leave this line as-is.

Reply:
Tested upgrading, if QWERTY was previously selected before upgrading, nothing will be selected after upgrading, the user will have to select the keyboard layout again for the app to work

Reply:
Yeah, we can't ship update like that as many people will have auto update on, they'll never see the release note.

Reply:
Gone ahead and renamed US QWERTY back to QWERTY, tested and it remains selected after update.